src/applications/vaos/services/location/transformers.js

```diff
@@ -311,3 +311,29 @@ export function setSupportedSchedulingMethods({
 export function transformFacilities(facilities) {
   return facilities.map(transformFacility);
 }
+
+export function transformCommunityProviders(providers) {
+  return providers.map(provider => {
+    return {
+      id: provider.uniqueId,
+      resourceType: 'Location',
```

Comment:
Transformed to Location because that's what Brad C said the backend would do in the future.